### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

**Comment by rtsisyk** [comment]
I am experimenting with moving this translation to Weblate. Please ping me via any channel if you want to help.

File: config.toml [hunk 1]
```diff
@@ -57,6 +57,24 @@ news = "Neuigkeiten"
 support-us = "Beitragen"
 token = "Token"
 
+[languages.fr]
+[languages.fr.translations]
+address = "Adresse"
+back = "Retour aux actualités"
+contact = "Contacts"
```
Comment (Jean-BaptisteC) [inline config.toml:64]:
```suggestion
contact = "Contact"
```

File: content/_index.fr.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+L'application est gratuite pour tous. Veuillez [faire un don](@/donate/index.md) pour nous soutenir !
+
+### Il y a plusieurs façons de faire un don :
+
+{{ donate_buttons(lang='fr') }}
+
+### Nos sponsors :
+
+Le FAI [Mythic Beasts](https://www.mythic-beasts.com/) [nous](https://www.mythic-beasts.com/blog/2021/10/06/improving-the-world-bit-by-expensive-bit/) fournit deux serveurs virtuels avec 400 TB/mois de bande passante gratuite pour aider nos utilisateurs à téléchargements et mises à jour de cartes.
```
Comment (Jean-BaptisteC) [inline content/_index.fr.md:93]:
```suggestion
Le FAI [Mythic Beasts](https://www.mythic-beasts.com/) [nous](https://www.mythic-beasts.com/blog/2021/10/06/improving-the-world-bit-by-expensive-bit/) fournit deux serveurs virtuels avec 400 TB/mois de bande passante gratuite pour aider nos utilisateurs à télécharger et mettre à jour les cartes.
```

File: content/donate/index.fr.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+### Crypto
+
+Veuillez envisager de faire un don annuel pour réduire les frais totaux. Les réseaux Algorand, Tron, Litecoin et Monero offrent des frais réduits, mais la situation change tous les jours.
+
+{{ crypto_table(lang='fr') }}
+
+### Autres moyens de donner
+
+Oui ! Il existe de nombreuses façons de soutenir Organic Maps. Veuillez consulter la page la page [Contribuer](@/support-us/index.md) pour plus de détails.
```
Comment (Jean-BaptisteC) [inline content/donate/index.fr.md:67]:
```suggestion
Oui ! Il existe de nombreuses façons de soutenir Organic Maps. Veuillez consulter la page [Contribuer](@/support-us/index.md) pour plus de détails.
```

File: content/support-us/index.fr.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  Chaînes manquantes dans l'interface de l'application.
+- Traduisez [App Store](https://github.com/organicmaps/organicmaps/tree/master/iphone/metadata/en-US)
+  et [Android](https://github.com/organicmaps/organicmaps/tree/master/android/src/google/play/listings/en-US)
+  dans votre langue.
+- [Traduisez](https://github.com/organicmaps/organicmaps.github.io) notre site web dans votre langue.
+- Rejoignez la communauté [OpenStreetMap](https://www.openstreetmap.org/about) et contribuez aux données cartographiques.
+- Corriger les villes rouges trouvées par notre [validateur de transport public](https://cdn.organicmaps.app/subway/), pour que les métros et les tramways fonctionnent dans l'application.
+- Soutenez les autres utilisateurs sur [GitHub](https://github.com/organicmaps/organicmaps/issues),
+  [Telegram](https://t.me/OrganicMaps) ([на русском](https://t.me/OrganicMapsRu)),
+  [[matrice]](https://matrix.to/#/#organicmaps:matrix.org),
```
Comment (Jean-BaptisteC) [inline content/support-us/index.fr.md:29]:
```suggestion
  [[Matrix]](https://matrix.to/#/#organicmaps:matrix.org),
```

**Review by Jean-BaptisteC — APPROVED** [review]
Some corrections :)

**Comment by rtsisyk** [comment]
=> https://hosted.weblate.org/projects/organicmaps/website/fr/

**Comment by rtsisyk** [comment]
I've triggered automatic translation via Deepl in Weblate. Weblate will get almost the same content in an hour or so.

**Comment by rtsisyk** [comment]
@Jean-BaptisteC I will integrate your changes manually soon.

File: config.toml [hunk 5]
```diff
@@ -57,6 +57,24 @@ news = "Neuigkeiten"
...
+donate = "Donner"
+home = "Page d'accueil"
+install-appgallery = "Installez Organic Maps à partir de l'AppStore"
```
Comment (Jean-BaptisteC) [inline config.toml:67]:
```suggestion
install-appgallery = "Installez Organic Maps depuis l'AppGallery"
```

File: config.toml [hunk 6]
```diff
@@ -57,6 +57,24 @@ news = "Neuigkeiten"
...
+install-appstore = "Installez Organic Maps à partir de l'AppStore"
```
Comment (Jean-BaptisteC) [inline config.toml:68]:
```suggestion
install-appstore = "Installez Organic Maps depuis l'AppStore"
```

File: config.toml [hunk 7]
```diff
@@ -57,6 +57,24 @@ news = "Neuigkeiten"
...
+install-googleplay = "Installez Organic Maps à partir de Google Play"
```
Comment (Jean-BaptisteC) [inline config.toml:69]:
```suggestion
install-googleplay = "Installez Organic Maps depuis Google Play"
```

File: content/_index.fr.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
+---
+title: "Organic Maps: cartes hors ligne et navigateur GPS"
+description: Cartes open source pour voyageurs, touristes, cyclistes et randonneurs.
+page_template: index.html
+sort_by: weight
+extra:
+  menu_title: Page d'accueil
+---
+
+**Organic Maps** est une application Android et iOS gratuite de cartes hors ligne pour les voyageurs, les touristes, les randonneurs et les cyclistes, basée sur des données provenant de la foule. **[OpenStreetMap](https://www.openstreetmap.org)**. Il s'agit d'un [fork](https://en.wikipedia.org/wiki/Fork_(software_development)) de l'application **Maps.me** (précédemment connue sous le nom de [**MapsWithMe**](https://en.wikipedia.org/wiki/Maps.me)), axé sur la confidentialité et maintenu par les mêmes personnes qui ont créé **MapsWithMe** en 2011.
```
Comment (Jean-BaptisteC) [inline content/_index.fr.md:10]:
```suggestion
**Organic Maps** est une application Android et iOS gratuite de cartes hors ligne pour les voyageurs, les touristes, les randonneurs et les cyclistes, basée sur des données provenant des contributeurs. **[OpenStreetMap](https://www.openstreetmap.org)**. Il s'agit d'un [fork](https://en.wikipedia.org/wiki/Fork_(software_development)) de l'application **Maps.me** (précédemment connue sous le nom de [**MapsWithMe**](https://en.wikipedia.org/wiki/Maps.me)), axé sur la confidentialité et maintenu par les mêmes personnes qui ont créé **MapsWithMe** en 2011.
```

File: content/_index.fr.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+<img src='/images/privacy/exodus.png' width='400'>
+
+Organic Maps ne demande pas de permissions excessives pour vous espionner:
+
+<img src='/images/privacy/om.jpg' width='400'>
+<img src='/images/privacy/mm.jpg' width='400'>
+
+Chez Organic Maps, nous pensons que la <b>vie privée est un droit humain fondamental</b> :
+
+• Organic Maps est un projet indépendant à code source ouvert dirigé par la communauté.
```
Comment (Jean-BaptisteC) [inline content/_index.fr.md:75]:
```suggestion
• Organic Maps est un projet indépendant open-source dirigé par la communauté.
```

**Review by Jean-BaptisteC — APPROVED** [review]
Some others corrections :)

CC @arnaudvergnet

**Comment by rtsisyk** [comment]
@organicmaps/translations-fr, please take a look at https://rt-french.organicmaps.pages.dev/fr/

File: content/_index.fr.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+**Organic Maps** est l'une des rares applications actuelles à prendre en charge 100 % des fonctionnalités sans connexion Internet active. Installez Organic Maps, téléchargez des cartes, jetez votre carte SIM (d'ailleurs, votre opérateur vous suit constamment), et partez pour un voyage hebdomadaire sur une seule charge de batterie sans aucun octet envoyé au réseau.
+
+{{ badges(lang='fr') }}
+
+{{ screenshot(src='/images/screenshots/hiking.jpg', alt='Hiking') }}
+{{ screenshot(src='/images/screenshots/prague.jpg', alt='Prague') }}
+{{ screenshot(src='/images/screenshots/search.jpg', alt='Offline Search') }}
+{{ screenshot(src='/images/screenshots/dark.jpg', alt='Navigation in dark mode') }}
+
+## Features
```
Comment (Jean-BaptisteC) [inline content/_index.fr.md:21]:
```suggestion
## Fonctionnalités
```

File: content/_index.fr.md [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+- Itinéraires cyclables, sentiers de randonnée et chemins de promenade.
+- Courbes de niveau, profils d'élévation, sommets et pentes.
+- Navigation à pied, à vélo et en voiture, virage par virage, avec guidage vocal.
+- Recherche rapide hors ligne sur la carte
+- Exportation et importation de signets aux formats KML/KMZ (GPX est [prévu](https://github.com/organicmaps/organicmaps/issues/624))
+- Mode sombre pour protéger vos yeux
+- Les pays et les régions ne prennent pas beaucoup de place
+- Gratuit et open-source
+
+## Why Organic?
```
Comment (Jean-BaptisteC) [inline content/_index.fr.md:35]:
```suggestion
## Pourquoi Organic?
```

File: content/_index.fr.md [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+
+Organic Maps est pur et biologique, fabriqué avec amour :
+
+- Respecte votre vie privée
+- Économise votre batterie
+- Pas de frais de données mobiles inattendus
+
+L'application Organic Maps est exempte de traceurs et d'autres éléments indésirables :
```
Comment (Jean-BaptisteC) [inline content/_index.fr.md:43]:
```suggestion
L'application Organic Maps est exempte de trackers et d'autres éléments indésirables :
```

File: content/_index.fr.md [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+- Pas de collecte de données
+- Pas d'appel au domicile
+- Pas d'inscription fastidieuse
+- Pas de tutoriels obligatoires
+- Pas de spam excessif
+- Pas de notifications push
+- Pas de logiciels malveillants
+- N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Entièrement bio
+
+Selon <a href='https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/'>Exodus Privacy Project</a> il n'y a aucun traceur et les autorisations requises sont minimales:
```
Comment (Jean-BaptisteC) [inline content/_index.fr.md:64]:
```suggestion
Selon <a href='https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/'>Exodus Privacy Project</a> il n'y a aucun tracker et les autorisations requises sont minimales:
```

File: content/_index.fr.md [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+<img src='/images/privacy/om.jpg' width='400'>
+<img src='/images/privacy/mm.jpg' width='400'>
+
+Chez Organic Maps, nous pensons que la <b>vie privée est un droit humain fondamental</b> :
+
+• Organic Maps est un projet indépendant open-source dirigé par la communauté.
+• Nous protégeons la vie privée des regards indiscrets des géants de la tech
+• Restez en sécurité, où que vous soyez
+
+Reject surveillance - embrace your freedom.
```
Comment (Jean-BaptisteC) [inline content/_index.fr.md:79]:
```suggestion
Rejeter la surveillance - embrasser votre liberté.
```

File: content/donate/index.fr.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+weight: 10
+extra:
+  menu_title: Faire un don
+  preview_image: donate/donate.png
+---
+
+L'application Organic Maps est _gratuite pour tout le monde_ grâce à vos **[donations][stripe]** :
+
+- Pas de publicité
+- Pas de traceurs
```
Comment (Jean-BaptisteC) [inline content/donate/index.fr.md:13]:
```suggestion
- Pas de trackers
```

File: content/donate/index.fr.md [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+- Pas d'inscription
+- Pas de notifications push
+- Source ouverte
```
Comment (Jean-BaptisteC) [inline content/donate/index.fr.md:16]:
```suggestion
- Open-source
```

**Review by Jean-BaptisteC — APPROVED** [review]
Somes improvements :)

**Comment by biodranik** [comment]
@rtsisyk this one is important.

**Pull request merged**